### PR TITLE
removed the tests as per Bug #1730292/#1321543

### PR DESCRIPTION
**Pull request by vijay8451**
- Removed the tests as per : https://bugzilla.redhat.com/show_bug.cgi?id=1730292 and it is closed as duplicate of https://bugzilla.redhat.com/show_bug.cgi?id=1321543

- Tests  are requires to remove as per https://bugzilla.redhat.com/show_bug.cgi?id=1321543#c22

### CODE REVIEW

**Comment by vijay8451** [comment]
- Fixed #7185

File: tests/foreman/ui/test_location.py [hunk 1]
```diff
@@ -147,6 +148,7 @@ def test_positive_update_user(session):
         assert user.login in loc_values['users']['resources']['unassigned']
 
 
+@skip_if_bug_open('bugzilla', 1730292)
```
Comment (ntkathole) [inline tests/foreman/ui/test_location.py]: Closed as dupe of another bug which is WONTFIX

Comment (vijay8451) [reply tests/foreman/ui/test_location.py]: yeah .. was checking the same ... looks modify the tests now :)

Comment (vijay8451) [reply tests/foreman/ui/test_location.py]: @ntkathole @omkarkhatavkar tests are removed and updated the same in PR too ... would ask a revisit again on it ...
I am resolving this one no more applicable.

**Comment by vijay8451** [comment]
@pondrejk can you review as you are the component owner around this PR.

**Comment by vijay8451** [comment]
@pondrejk Yeah we tried by raising new bug[https://bugzilla.redhat.com/show_bug.cgi?id=1730292 ] so not very sure if try once more would help much ..

yeah that make sense to remove the assertion and check with 'visible in the user list'
Let me close this PR and raise another one.

**Pull request closed without merging**